### PR TITLE
[Merged by Bors] - chore(Mathlib/NumberTheory/NumberField/*): rename completion -> Completion

### PR DESCRIPTION
**Pull request by kbuzzard**
We have `UniformSpace.Completion` so it should surely also be `AbsoluteValue.Completion v : Type*` and `NumberField.InfinitePlace.Completion v : Type*` (currently they are both `completion` and both types). 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/Normed/Ring/WithAbs.html#AbsoluteValue.completion

https://leanprover-community.github.io/mathlib4_docs/Mathlib/NumberTheory/NumberField/Completion.html#NumberField.InfinitePlace.completion

### CODE REVIEW

File: Mathlib/Analysis/Normed/Ring/WithAbs.lean [hunk 1]
```diff
@@ -141,18 +141,18 @@ open WithAbs
 variable {K : Type*} [Field K] (v : AbsoluteValue K ℝ)
 
 /-- The completion of a field with respect to a real absolute value. -/
-abbrev completion := UniformSpace.Completion (WithAbs v)
+abbrev Completion := UniformSpace.Completion (WithAbs v)
```
Comment (eric-wieser) [inline Mathlib/Analysis/Normed/Ring/WithAbs.lean:144]: Strictly we should add a `deprecated alias` here for the `completion` spelling. No need to do it for all the other lemmas.

File: Mathlib/NumberTheory/NumberField/Completion.lean [hunk 2]
```diff
@@ -120,48 +120,48 @@ theorem subfield_ne_real_of_isComplex {v : InfinitePlace K} (hv : IsComplex v) :
...
+/-- If `v` is a complex infinite place, then the embedding `v.Completion →+* ℂ` is bijective. -/
 theorem bijective_extensionEmbedding_of_isComplex {v : InfinitePlace K} (hv : IsComplex v) :
     Function.Bijective (extensionEmbedding v) :=
   ⟨(extensionEmbedding v).injective, surjective_extensionEmbedding_of_isComplex hv⟩
 
-/-- The ring isomorphism `v.completion ≃+* ℂ`, when `v` is complex, given by the bijection
-`v.completion →+* ℂ`. -/
+/-- The ring isomorphism `v.Completion ≃+* ℂ`, when `v` is complex, given by the bijection
+`v.Completion →+* ℂ`. -/
 def ringEquiv_complex_of_isComplex {v : InfinitePlace K} (hv : IsComplex v) :
```
Comment (eric-wieser) [inline Mathlib/NumberTheory/NumberField/Completion.lean:137]: This is also misnamed, but that's a problem for another PR, not this one.

Comment (kbuzzard) [reply Mathlib/NumberTheory/NumberField/Completion.lean:139]: The logic here is that an infinite place `v` of a number field is either real (`NumberField.InfinitePlace.IsReal v : Prop`) or complex (`...IsComplex v`). In the real case,  `v.Completion` is (canonically) isomorphic to the reals, and in the complex case, `v.Completion` is (non-canonically) isomorphic to the complexes (the issue is that you can't distinguish the isomorphism and its complex conjugate). So "if the place is complex, we're isomorphic to the complexes". Do you still think it's misnamed @eric-wieser ?

Comment (kbuzzard) [reply Mathlib/NumberTheory/NumberField/Completion.lean:139]: #19659

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

**Comment by riccardobrasca** [comment]
Can you please fix the conflict? Thanks!

bors d+

**Comment by kbuzzard** [comment]
Sorry for the delay, I'm travelling right now. Let's see if I can get this compiling again.

**Comment by kbuzzard** [comment]
bors merge

**Pull request closed without merging**